src/components/AuthAndFrame.tsx

```diff
@@ -1,27 +1,16 @@
-import React from "react";
+import React, { useState } from "react";
 import RequireAuth from "./RequireAuth";
 import Footer from "./Footer";
 import Navbar from "./Navbar";
 import { NavProps } from "./ListItems";
-import { getCurrentUser } from "../services/login";
+import { useDispatch, useSelector } from "react-redux";
+import { currentUserSelector, getCurrentUser } from "../store/auth/authSlice";
 
 interface AuthAndFrameProps extends NavProps {
   children: JSX.Element;
 }
 
 export const AuthAndFrame: React.FC<AuthAndFrameProps> = (props) => {
-  const [currentUser, setCurrentUser] = React.useState<string | null>(null);
```

Comment:
Acutally, the user info is used for navbar here, but we can reimplement it when it is in need.